README.md

```diff
@@ -277,6 +277,43 @@ it is not possible to contain a bind-mounted subdirectory in a separately bind-m
 directory. You could see these sources by setting `SERIALBOX_OUTDIR` to a different
 directory (it is `/FV3` by default) that you have bind-mounted in to the container.
 
+# The Python wrapper
+
+The `FV3/wrapper` subdirectory contains a python wrappper that can be used to
+call the fortran model for interactive ML. It's usage docs can be seen [here](TODO/add/path).
+
+To build the wrapper some additional python requirements must be available in the environment:
+- fv3config
+- pace-util
+- numpy
+- pyyaml
+- xarray
+- cython
+- mpi4py
+
+The [nix environment][#developing] is setup automatically with these
+dependencies, and is the recommend development environment for iterative
+development (edit/build/test) of the wrapper. In other environments, e.g.
+HPC/DOCKER these builds dependencies will need to be manually installed.
+
+Once the dependencies are installed the wrapper and fv3.exe can be built like
+this
+
+    AI2_WRAPPER=Y make -C FV3
+
+To test the wrapper
+
+    # needed to import `fv3gfs-wrapper`
+    export PYTHONPATH=$(pwd)/FV3/wrapper:$PYTHONPATH
```

Comment:
Thanks for the docs here! I have a high level request. Can you switch things around so first the instructions say how to build and install the wrapper (using the pip install command below), then how to test the wrapper if installed this way (just `make test_wrapper`), then how to set up a "development mode" for the wrapper (altering the python path if really necessary, or some kind of `pip install -e`)?

It may be nice to have a make target to build the wrapper wheel and install it. This may be the normal way we'd want to update the code in development mode, since this project is mainly compiled.

Is it not possible to `pip install -e` the wrapper? Having to modify the PYTHONPATH to run this makes me nervous.

Reply:
I changed the order here: ba220c0. Is that what you meant?

> Is it not possible to pip install -e the wrapper?

Umm. `pip install -e` would only work if invoked with all the same make-file target info as `make wrapper_build`. Would could make a makefile target in `FV3/makefile` to setup the development environment if PYTHONPATH is not a good solution. What's the downside of setting PYTHONPATH though? In some ways it's cleaner since it doesn't modify the filesystem and actually uses the installed versions of e.g. numpy. `pip install -e` actually downloads it's own potentially different version of `numpy`, something that has bit us several times in fv3net.

Reply:
Looks good as it is now.